app/src/main/java/org/mozilla/tv/firefox/pocket/PocketEndpoint.kt

```diff
@@ -37,26 +37,26 @@ object PocketEndpoint {
 
     /** @return The global video recommendations or null on error; the list will never be empty. */
     @AnyThread // via PocketEndpointRaw.
-    suspend fun getRecommendedVideos(): List<PocketVideo>? {
+    suspend fun getRecommendedVideos(): List<PocketFeedItem.Video>? {
         val videosJSON = PocketEndpointRaw.getGlobalVideoRecommendations() ?: return null
         return convertVideosJSON(videosJSON)
     }
     /** Developer Ergonomics
      * @return A placeholder list of videos when Pocket API key is not provided. */
-    fun getPlaceholderVideos(): List<PocketVideo>? {
-        val placeholderList = mutableListOf<PocketVideo>()
+    fun getPlaceholderVideos(): List<PocketFeedItem.Video>? {
```

Comment:
I don't think this function is needed anymore since it is in the VM and not being used anywhere.

Reply:
Good catch!  I'll pull that out.